ext/opentelemetry-ext-otcollector/src/opentelemetry/ext/otcollector/trace_exporter/__init__.py

```diff
@@ -94,10 +94,9 @@ def translate_to_collector(spans: Sequence[Span]):
     for span in spans:
         status = None
         if span.status is not None:
-            status = trace_pb2.Status(
-                code=span.status.canonical_code.value,
-                message=span.status.description,
-            )
+            status = trace_pb2.Status(code=span.status.canonical_code.value,)
+            if isinstance(span.status.description, str):
```

Comment:
What is the exact purpose of this check? Avoiding to set message to None or checking the type of the description?

We should avoid the second case in the exporters, here you are only checking a single field, if we apply the same logic all the fields in the span / attributes should be checked as well. I think we can be confident enough that it's going to be an `str` with the changes you did to the `Status` constructor.

Reply:
You're right, this is no longer needed here.